site/src/pages/api/rewrites/sandboxed-block-demo.api.ts

```diff
@@ -142,8 +143,11 @@ const handler: NextApiHandler = async (req, res) => {
       
           const entryPoint = blockType.entryPoint.toLocaleLowerCase();
           
-          const rawBlockSource = source;
-          const blockExport = entryPoint === "html" ? rawBlockSource : findBlockExport(loadCjsFromSource(rawBlockSource));
+          let blockExport = findBlockExport(loadCjsFromSource(source));
+          // @todo make this unnecessary
```

Comment:
Do you have thoughts about how best to achieve this?

A couple:
1. Make import in the template the thing you've written here
2. List `@blockprotocol/graph` as a peerDependency, in which case it'll become listed in `metadata.externals` and loaded like the others above